contrib/src/main/java/openconsensus/contrib/metrics/runtime/GarbageCollector.java

```diff
@@ -0,0 +1,63 @@
+package openconsensus.contrib.metrics.runtime;
```

Comment:
Nit: license header. Also probably better to split JVM metrics to another PR.

Reply:
JVM metrics are an example of usage to prove the new design choice. Also the contrib package in this repo is mostly to prove what we can build on top of the API. We need tests and better review before we release that.